homeassistant/components/abode/config_flow.py

```diff
@@ -24,6 +26,7 @@ def __init__(self):
         self.data_schema = {
             vol.Required(CONF_USERNAME): str,
             vol.Required(CONF_PASSWORD): str,
+            vol.Optional(CONF_POLLING, default=False): bool,
```

Comment:
This option should be part of the options flow, since it's optional and isn't needed to set up the connection correctly.

https://developers.home-assistant.io/docs/en/config_entries_options_flow_handler.html

Reply:
We can do this in a follow up PR. For now we can just keep the option in config yaml.

Reply:
The issue I ran into is when using the configuration.yaml file and `polling` is set to `True`, we run into an error with config_flow.py. My understanding is from config_flow.py, the `async_step_import` method handles the configuration.yaml data but right now, it's simply being passed to `async_step_user`:

`return await self.async_step_user(import_config)`

Within `async_step_user`, I don't have any variables defined for `polling` since it's not a part of the UI config flow, so it causes an error. I'm pretty sure there's a simple way to do this but I spent a few hours last night and couldn't figure it out. Basically I need a way to default `polling` to `False` unless it's passed in from `async_step_import`.

Reply:
And of course after 5 mins looking at it again, I think I figured out a simple solution. Within the `async_step_user` of the flow handler:

```
        if CONF_POLLING in user_input:
            polling = user_input[CONF_POLLING]
        else:
            polling = False
```